hazelcast/invocation.py

```diff
@@ -261,7 +261,7 @@ def _register_backup_listener(self):
         self._listener_service.register_listener(request,
                                                  codec.decode_response,
                                                  lambda reg_id: None,
-                                                 lambda m: codec.handle(m, self._backup_event_handler))
+                                                 lambda m: codec.handle(m, self._backup_event_handler)).result()
```

Comment:
Is it ok that this call is blocking?

Reply:
`HazelcastClient()` call is also blocking. So, I think that shouldn't be a problem

Reply:
I see. Thanks for the clarification.